docs/troubleshooting-oauth2.md

```diff
@@ -23,119 +23,116 @@ limitations under the License.
 10. Search for the token attribute `scope` in the tokens' payload or for the value configured in `auth_oauth2.additional_scopes_key`, if any.
 11. Once you found the appropriate token's scope attribute, find within the attribute's value any of the scopes listed above. If [auth_oauth2.scope_prefix](./oauth2#scope-prefix) is used, it must be taken into account: the scopes will be named like  `myprefix_tag:administrator`. If [scope aliases](./oauth2-examples#using-scope-aliases) are used, find the scope alias that maps to one of the scopes listed above
 
 
-### OpenId Discovery Endpoint not Reachable Due to a Certificate SAN (or CN) and Target Hostname Mismatch {#openid-discovery-endpoint-san-cn-mismatch}
+### OpenId Discovery endpoint unreachable due to bad certificate {#openid-discovery-endpoint-bad-certificate}
 
-This issue is not necessarily specific to the management UI, it may also occur when an application is authenticating via one of the messaging protocols.
-It occurs when RabbitMQ has to download the OpenID Connect configuration via the URL configured in `auth_oauth2.issuer` and the certificate used by the issuer uses a wildcard certificate.
-This means that the certificate's SAN (Subject Alternative Name) or CN certificate attributes does not match the issuer's domain name exactly.
-This is a very common situation in SaaS deployments.
+This issue is not necessarily specific to the management UI, it may also occur when an application is authenticating via one of the messaging protocols. It occurs when RabbitMQ has to download the OpenId Connect configuration via the url configured in `auth_oauth2.issuer` and the certificate used by the issuer uses a wildcard certificate. This means that the certificate's CN attribute does not match exactly the issuer's domain name. This is very common on SaaS deployments.
 
-#### Steps to reproduce the issue
+#### Steps to reproduce
 
 1. Open the root URL of the management UI in the browser.
 2. Click on "Click here to login".
 3. Enter your credentials.
-4. You are redirected back to RabbitMQ but with an error "No Authorized"
+4. You are redirected back to RabbitMQ but with an error "No Authorized".
 
-#### Troubleshoot the issue
+#### Troubleshooting
 
-1. Access RabbitMQ [logs](./logging)
-2. Look up for `{bad_cert,hostname_check_failed}`
-3. Look for an entry it says that RabbitMQ tried to contact the url found in `auth_oauth2.issuer` and the issuer presented a wildcard certificate
-4. To fix this issue, add the following line to your RabbitMQ configuration `auth_oauth2.https.hostname_verification = wildcard`. Or `auth_oauth2.oauth_providers.<my_oauth_provider_name>.https.hostname_verification = wildcard` if you are using multiple OAuth providers
+1. Access RabbitMQ logs.
+2. Look for `{bad_cert,hostname_check_failed}`.
+3. If you find an entry it means that RabbitMQ tried to contact the url found in `auth_oauth2.issuer` and the issuer presented a wildcard certificate.
```

Comment:
url or URL? keep consistent through, check existing OSS RMQ docs to see what is used consistently and continue with that, thanks

Reply:
add a comma after "If you find and entry,...."